src/utils/dashboardUtilFunctions.js

```diff
@@ -0,0 +1,401 @@
+import uuid from 'uuid';
+import { mergeWith, isArray } from 'lodash';
+
+export const COLORS_LEVEL_1 = [
+  '#D4D4D4',
+  '#057EBD',
+  '#0C3151',
+  '#F78F49',
+  '#79287C',
+  '#7CC242',
+  '#61479D',
+];
+
+export const COLORS_LEVEL_2 = [
+  '#F78F49',
+  '#79287C',
+  '#7CC242',
+  '#61479D',
+  '#D4D4D4',
+  '#057EBD',
+  '#0C3151',
+];
+
+const NOT_PROVIDED = 'Not Specified';
+/*
+ Group : GroupName that will show on the page as category
+ field: API return field name
+    eg: {
+          gender : male
+          cases: 123
+        }
+        gender is the field
+  api : API that we are using to get data.
+  dtatfield: datatable field that related
+  show: control show this category on the page or not
+
+*/
+
+export const unselectFilters = (filtersObj) => filtersObj.map((filterElement) => ({
+  groupName: filterElement.groupName,
+  name: filterElement.name,
+  datafield: filterElement.datafield,
+  isChecked: false,
+}));
+
+// eslint-disable-next-line consistent-return
+function customizer(objValue, srcValue) {
+  if (isArray(objValue)) {
+    return objValue.concat(srcValue);
+  }
+}
+
+export function getStatDataFromDashboardData(dashboardData, stats) {
```

Comment:
can you write function comments
/**
 * Performs this operation
 *
 * @param {number} x 
 * @param {number} n 
 * @return {number} x r.
 */